### PR TITLE
Implement slash commands for definitions and explanations of common terms

### PR DESCRIPTION
**Pull request by Pixxel123**
This PR adds slash commands which can be called to offer quick explanations about common terms and issues.

Of course, these are still a draft and will be passed through the community for improving explanations, definitions, etc.

In this first iteration, the terms available are:
- Explanation of shaders and shader caches
- Explanation of PPTC
- How to get log files
- What keys are and a link to the dumping guide
- What firmware is and a link to the dumping guide
- Getting basic support

Some examples of the commands:
image
image
image
image
image

As usual, suggestions are very much welcomed!

- [x] Explain FIFO
- [ ] Explain installing and using mods (do we want to wait for mod manager to be merged?)
- [x] Explain LDN
- [x] Add note about not using third-party shaders
- [x] Explain using Amiibos
- [ ] Explain resolution scaler?
- [x] Explain Docked/Handheld modes?
- [ ] Explain motion controls

### CODE REVIEW

File: robocop_ng/cogs/explain.py [hunk 1]
```diff
@@ -1,49 +1,120 @@
...
+            },
+            "support": {
+                "body_text": f"""Please use <#{config.bot_log_allowed_channels["support"]}>, <#{config.bot_log_allowed_channels['patreon-support']}> or <#{config.bot_log_allowed_channels['linux-master-race']}> channels to get help.
+                                The easiest way to get help is to post a log file, since that will have information on your specs and hardware:
+                                \nLog files are saved in are saved in the `Logs` folder.
+                                Access this by doing `File > Open Log Folder` in Ryujinx, or by going to the folder directly where your `Ryujinx.exe` (Windows) or `Ryujinx` binary (Linux) is located.
+                                The last 3 logs from the last 3 boots of Ryujinx will be available. For support purposes, you'll usually want to drag and drop the largest one into Discord.
+                                \nIf you are unable to provide a log file, please provide your basic specs including: Ryujinx version, CPU model, GPU model and RAM amount, as well as **the name of the game and the version of the game you're having issues with.**
```
Comment (choppymaster) [inline robocop_ng/cogs/explain.py:40]: Any game mod things needed?

File: robocop_ng/cogs/explain.py [hunk 2]
```diff
@@ -1,49 +1,120 @@
...
         embed = Embed(
             title=self.explanations["pptc"]["title"],
-            description=self.explanations["pptc"]["description"],
-            colour=Colour(0x4A90E2),
+            description=self.explanations["pptc"]["body_text"],
+            colour=self.ryujinx_blue,
+        )
+        await ctx.send(embed=embed)
+
+    @cog_ext.cog_slash(name="logs", description="How to get Ryujinx log files.")
```
[the remark this answers is not in the document]
Comment (Pixxel123) [reply robocop_ng/cogs/explain.py:88]: I don't think that distinction is necessary, it's telling you _how_ to get log files, not explaining how they work.

File: robocop_ng/cogs/explain.py [hunk 3]
```diff
@@ -1,49 +1,120 @@
...
+                                The last 3 logs from the last 3 boots of Ryujinx will be available. For support purposes, you'll usually want to drag and drop the largest one into Discord.""",
+                "title": "How to get log files",
+            },
+            "support": {
+                "body_text": f"""Please use <#{config.bot_log_allowed_channels["support"]}>, <#{config.bot_log_allowed_channels['patreon-support']}> or <#{config.bot_log_allowed_channels['linux-master-race']}> channels to get help.
+                                The easiest way to get help is to post a log file, since that will have information on your specs and hardware:
+                                \nLog files are saved in are saved in the `Logs` folder.
+                                Access this by doing `File > Open Log Folder` in Ryujinx, or by going to the folder directly where your `Ryujinx.exe` (Windows) or `Ryujinx` binary (Linux) is located.
+                                The last 3 logs from the last 3 boots of Ryujinx will be available. For support purposes, you'll usually want to drag and drop the largest one into Discord.
```
Comment (choppymaster) [inline robocop_ng/cogs/explain.py:40]: Change `drag and drop` to `upload`?

File: robocop_ng/cogs/explain.py [hunk 4]
```diff
@@ -1,49 +1,120 @@
...
+            description=self.explanations["logs"]["body_text"],
+            colour=self.ryujinx_blue,
+        )
+        await ctx.send(embed=embed)
+
+    @cog_ext.cog_slash(name="support", description="How to get help with Ryujinx.")
+    async def get_support(self, ctx: SlashContext):
+        embed = Embed(
+            title=self.explanations["support"]["title"],
+            description=self.explanations["support"]["body_text"],
```
Comment (choppymaster) [inline robocop_ng/cogs/explain.py:101]: I will recommend moving the embed generating to a seperate function as all the commands generate the embed by the same. Like:
```py
def generate_embed(self, cmd_name):
  return Embed(
                title=self.explanations[cmd_name]["title"],
                description=self.explanations[cmd_name]["body_text"],
                colour=self.ryujinx_blue
              )
```

**Comment by marysaka** [comment]
Closing as stale.

**Pull request closed without merging**